packages/pn-personafisica-webapp/src/component/Contacts/DigitalContactElem.tsx

```diff
@@ -39,6 +39,7 @@ type Props = {
     size: 'auto' | 'variable';
   }>;
   recipientId: string;
+  isVerifyingAddress?: boolean;
```

Comment:
I shouldn't add new props to this component, but I should conditionally render the component where it is used